synapse/federation/federation_client.py

```diff
@@ -553,7 +553,7 @@ def make_membership_event(
         Note that this does not append any events to any graphs.
 
         Args:
-            destinations (str): Candidate homeservers which are probably
+            destinations (Iterable[unicode]): Candidate homeservers which are probably
```

Comment:
unicode doesn't really exist any more in python 3:
```suggestion
            destinations (Iterable[str]): Candidate homeservers which are probably
```